assets/js/WfParameter.js

```diff
@@ -365,7 +365,8 @@ export default class WfParameter extends React.PureComponent {
   render() {
     const { id_name, name, type, visible_if, visible_if_not } = this.props.p.parameter_spec
 
-    if (!this.props.p.visible) {
+    // TODO: force display of 'colnames' for now since it will completely replace 'colselect' eventually
```

Comment:
This comment says the opposite of what we want. Replace `TODO` with `TODO: delete the 'colnames' check` and then keep the rest of the comment.